cli/command/container/update.go

```diff
@@ -56,7 +56,7 @@ func NewUpdateCommand(dockerCli command.Cli) *cobra.Command {
 	}
 
 	flags := cmd.Flags()
-	flags.Uint16Var(&options.blkioWeight, "blkio-weight", 0, "Block IO (relative weight), between 10 and 1000, or 0 to disable (default 0)")
+	flags.Uint16Var(&options.blkioWeight, "blkio-weight", 0, `Block IO (relative weight), between 10 and 1000, or 0 to disable (default "0")`)
 	flags.Int64Var(&options.cpuPeriod, "cpu-period", 0, "Limit CPU CFS (Completely Fair Scheduler) period")
```

Comment:
Not something new, but my eye dropped on these in the new tables; we have various options where `0` (or `0 sec`) is the default.

In the case above, "0" is explained (so "OK(ish)") but in various other cases, `0` very likely means "use default" (e.g. `cpu-period`). We should review which ones need "0" to be shown as default value, and for which ones we should (ideally) _not_ print the default (as the default basically means "not set").